doc/libpmemobj/pmemobj_open.3.md

```diff
@@ -70,7 +70,7 @@ used as a check when _UW(pmemobj_open) is called. The layout name, including
 the terminating null byte ('\0'), cannot be longer than **PMEMOBJ_MAX_LAYOUT**
 as defined in **\<libpmemobj.h\>**. A NULL *layout* is equivalent
 to using an empty string as a layout name. *mode* specifies the permissions to
-use when creating the file, as described by **creat**(2). The memory pool file
+use when creating the file, as described by **create**(2). The memory pool file
```

Comment:
``
    Ken Thompson was once asked what he would do differently if he were redesigning the UNIX system. His reply: "I'd spell creat with an e."
        Kernighan, Brian W.; Pike, Rob (1984). The UNIX programming environment. Prentice-Hall. ISBN 0139376992. OCLC 10269821., p. 204.``